pyseir/cli.py

```diff
@@ -66,6 +66,8 @@ def entry_point():
             structlog.stdlib.add_log_level,  # required before SentryProcessor()
             # sentry_sdk creates events for level >= ERROR and keeps level >= INFO as breadcrumbs.
             SentryProcessor(level=logging.INFO),
+            structlog.processors.TimeStamper(fmt="iso"),
```

Comment:
Please add to https://github.com/covid-projections/covid-data-model/blob/master/run.py#L34-L38 too. My mistake for not factoring it out.

Reply:
good call! I hadn't thought about the other entry point. added in the latest commit.